### PR TITLE
[EG] Update docs

### PR DESCRIPTION
**Pull request by l0lawrence**
Removing http references from docstrings

### CODE REVIEW

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 1]
```diff
@@ -297,7 +297,7 @@ namespace Microsoft.EventGrid {
   // Acknowledge Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:acknowledge&apiVersion={apiVersion}
 
-  @doc("Acknowledge batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully acknowledged lockTokens, along with other failed lockTokens with their corresponding error information. Successfully acknowledged events will no longer be available to any consumer.")
+  @doc("Acknowledge batch of Cloud Events. The response body will include the set of successfully acknowledged lockTokens, along with other failed lockTokens with their corresponding error information. Successfully acknowledged events will no longer be available to any consumer.")
```
Comment (lmazuel) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:300]:
```suggestion
  @doc("Acknowledge batch of Cloud Events. The response will include the set of successfully acknowledged lockTokens, along with other failed lockTokens with their corresponding error information. Successfully acknowledged events will no longer be available to any consumer.")
```

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 2]
```diff
@@ -312,7 +312,7 @@ namespace Microsoft.EventGrid {
   // Release Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:release?api-version={apiVersion}&releaseDelayInSeconds={delay}
 
-  @doc("Release batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully released lockTokens, along with other failed lockTokens with their corresponding error information.")
+  @doc("Release batch of Cloud Events. The response body will include the set of successfully released lockTokens, along with other failed lockTokens with their corresponding error information.")
```
Comment (lmazuel) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:315]:
```suggestion
  @doc("Release batch of Cloud Events. The response will include the set of successfully released lockTokens, along with other failed lockTokens with their corresponding error information.")
```

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 3]
```diff
@@ -333,7 +333,7 @@ namespace Microsoft.EventGrid {
   // Reject Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:reject?api-version={apiVersion}
 
-  @doc("Reject batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully rejected lockTokens, along with other failed lockTokens with their corresponding error information.")
+  @doc("Reject batch of Cloud Events. The response body will include the set of successfully rejected lockTokens, along with other failed lockTokens with their corresponding error information.")
```
Comment (lmazuel) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:336]:
```suggestion
  @doc("Reject batch of Cloud Events. The response will include the set of successfully rejected lockTokens, along with other failed lockTokens with their corresponding error information.")
```

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 4]
```diff
@@ -349,7 +349,7 @@ namespace Microsoft.EventGrid {
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:renewLock?api-version={apiVersion}
   @added(ServiceApiVersions.v2023_10_01_preview)
   @removed(ServiceApiVersions.v2023_11_01)
-  @doc("Renew lock for batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully renewed lockTokens, along with other failed lockTokens with their corresponding error information.")
+  @doc("Renew lock for batch of Cloud Events. The response body will include the set of successfully renewed lockTokens, along with other failed lockTokens with their corresponding error information.")
```
Comment (lmazuel) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:352]:
```suggestion
  @doc("Renew lock for batch of Cloud Events. The response will include the set of successfully renewed lockTokens, along with other failed lockTokens with their corresponding error information.")
```

**Comment by billwert** [comment]
It turns out in Java I will probably override a lot of these anyway to have better samples and things. More generally, though, do we want to include any explanation here of what these do? For example we could explain release semantic a bit.

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 5]
```diff
@@ -232,7 +232,7 @@ namespace Microsoft.EventGrid {
   // Publish Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}:publish?api-version={apiVersion}}
 
-  @doc("Publish Single Cloud Event to namespace topic. In case of success, the server responds with an HTTP 200 status code with an empty JSON object in response. Otherwise, the server can return various error codes. For example, 401: which indicates authorization failure, 403: which indicates quota exceeded or message is too large, 410: which indicates that specific topic is not found, 400: for bad request, and 500: for internal server error. ")
+  @doc("Publish a Single Cloud Event to a namespace topic.")
```
Comment (jsquire) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:235]:
```suggestion
  @doc("Publish a single Cloud Event to a namespace topic.")
```

Not Sure why Single was Capitalized.

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 6]
```diff
@@ -251,7 +251,7 @@ namespace Microsoft.EventGrid {
     }
   >;
 
-  @doc("Publish Batch Cloud Event to namespace topic. In case of success, the server responds with an HTTP 200 status code with an empty JSON object in response. Otherwise, the server can return various error codes. For example, 401: which indicates authorization failure, 403: which indicates quota exceeded or message is too large, 410: which indicates that specific topic is not found, 400: for bad request, and 500: for internal server error. ")
+  @doc("Publish a Batch of Cloud Events to a namespace topic.")
```
Comment (jsquire) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:254]:
```suggestion
  @doc("Publish a batch of Cloud Events to a namespace topic.")
```

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 7]
```diff
@@ -274,7 +274,7 @@ namespace Microsoft.EventGrid {
   // Receive Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:receive?api-Version={apiVersion}&maxWaitTime=60&maxEvents={maxEvents}
 
-  @doc("Receive Batch of Cloud Events from the Event Subscription.")
+  @doc("Receive a Batch of Cloud Events from the Event Subscription.")
```
Comment (jsquire) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:277]:
```suggestion
  @doc("Receive a batch of Cloud Events from a subscription.")
```

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 8]
```diff
@@ -296,7 +296,7 @@ namespace Microsoft.EventGrid {
   // Acknowledge Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:acknowledge&apiVersion={apiVersion}
 
-  @doc("Acknowledge batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully acknowledged lockTokens, along with other failed lockTokens with their corresponding error information. Successfully acknowledged events will no longer be available to any consumer.")
+  @doc("Acknowledge a Batch of Cloud Events. The response will include the set of successfully acknowledged lockTokens, along with other failed lockTokens with their corresponding error information. Successfully acknowledged events will no longer be available to any consumer.")
```
Comment (jsquire) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:299]:
```suggestion
  @doc("Acknowledge a batch of Cloud Events. The response will include the set of successfully acknowledged lock tokens, along with other failed lock tokens with their corresponding error information. Successfully acknowledged events will no longer be available to be received by any consumer.")
```

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 9]
```diff
@@ -311,7 +311,7 @@ namespace Microsoft.EventGrid {
   // Release Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:release?api-version={apiVersion}&releaseDelayInSeconds={delay}
 
-  @doc("Release batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully released lockTokens, along with other failed lockTokens with their corresponding error information.")
+  @doc("Release a Batch of Cloud Events. The response will include the set of successfully released lockTokens, along with other failed lockTokens with their corresponding error information. Successfully released events will be available for other consumers.")
```
Comment (jsquire) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:314]:
```suggestion
  @doc("Release a batch of Cloud Events. The response will include the set of successfully released lock tokens, along with other failed lock tokens with their corresponding error information. Successfully released events can be received by consumers.")
```

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 10]
```diff
@@ -332,7 +332,7 @@ namespace Microsoft.EventGrid {
   // Reject Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:reject?api-version={apiVersion}
 
-  @doc("Reject batch of Cloud Events. The server responds with an HTTP 200 status code if the request is successfully accepted. The response body will include the set of successfully rejected lockTokens, along with other failed lockTokens with their corresponding error information.")
+  @doc("Reject a Batch of Cloud Events. The response will include the set of successfully rejected lockTokens, along with other failed lockTokens with their corresponding error information. Successfully rejected events will be deadlettered and no longer available to any consumer.")
```
Comment (jsquire) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:335]:
```suggestion
  @doc("Reject a batch of Cloud Events. The response will include the set of successfully rejected lock tokens, along with other failed lock tokens with their corresponding error information. Successfully rejected events will be dead-lettered and can no longer be received by a consumer.")
```

**Review by lmazuel — APPROVED** [review]

**Pull request merged**